README.md

```diff
@@ -4,6 +4,34 @@
 ## Source Control Infrastructure
 For this project, we will be Git for source control with a feature branch style of source control. 
 
+### Step-To-Step Git Instructions to Make Changes
+1. Supposed you have already had your git setup properly
+2. Make sure your local repo is updated with master.
+
+    ` git pull origin master`
+3. Create and move to a **new branch**
+
+    ` git branch new-sample-branch`
+
+    ` git checkout new-sample-branch`
+4. Now that you are on your own feature branch, make whatever changes you want!
+5. Once you complete your feature, **add, commit and push** your change to Github. If you do everything correctly, you would see your change in Github branch: features/new-sample-branch
```

Comment:
in this case, the branch would be called new-sample-branch, not features/new-sample-branch